app/pages/_app.tsx

```diff
@@ -27,6 +32,24 @@ function MyApp({ Component, pageProps }: AppProps) {
     setError(null);
   }, [Component]);
 
+  useMemo(() => {
```

Comment:
I would add a small comment about the caveat that the use of `useMemo` here means the feature toggle in growthbook means the change won't be visible for a user until they refresh the page